packages/email-components/src/components/Heading/Heading.ts

```diff
@@ -32,20 +32,36 @@ export default class Heading extends BodyComponent {
   static allowedAttributes = {
     align: 'enum(left,right,center)',
     elementType: 'enum(div,h1,h2,h3,h4,p,span)',
+    'font-family': 'string',
+    'font-size': 'unit(px)',
+    'line-height': 'unit(px,%,)',
     size: 'enum(medium,large)',
     title: 'string',
   };
 
   static defaultAttributes = {
     align: 'left',
     elementType: 'div',
-    size: 'medium',
+    'font-family': 'Arial, -apple-system, sans-serif',
+    'font-size': '16px',
+    'line-height': '1.5',
+    size: 'large',
```

Comment:
⚠️ This could be a BC in the language of the semver. Do we really want the default value `large`?

Reply:
Thanks for the review you have right the default value has to be `medium`